code/modules/events/fugitive_spawning.dm

```diff
@@ -12,27 +12,29 @@
 	fakeable = FALSE
 
 /datum/round_event/ghost_role/fugitives/spawn_role()
-	var/turf/landing_turf = pick(GLOB.xeno_spawn)
+	var/list/possible_spawns = list()//Some xeno spawns are in some spots that will instantly kill the refugees, like atmos
+	for(var/turf/X in GLOB.xeno_spawn)
+		if(istype(get_area(X), /area/maintenance))
+			possible_spawns += X
```

Comment:
Is there always one, should probably fail if there isn't. And since you know these are turfs you don't need to use get_area.

Reply:
I think you meant X.loc in both lines here

Reply:
if X.loc would be the turf's area than the above check needs to be that, but this line at least should stay because it's adding a turf to the possible spawns

Reply:
turf's loc is area